hudi-utilities/src/main/java/org/apache/hudi/utilities/transform/SqlFileBasedTransformer.java

```diff
@@ -72,25 +71,21 @@ public Dataset<Row> apply(
       final TypedProperties props) {
 
     final String sqlFile = getStringWithAltKeys(props, SqlTransformerConfig.TRANSFORMER_SQL_FILE);
-    if (null == sqlFile) {
-      throw new HoodieTransformException(
-          "Missing required configuration : (" + SqlTransformerConfig.TRANSFORMER_SQL_FILE.key() + ")");
-    }
```

Comment:
Could you add tests on both transformers that the exception is thrown if the config is missing?

Reply:
Added check for `TestSqlQueryBasedTransformer`, in `TestSqlFileBasedTransformer` it is already written:  
```
@Test
  public void testSqlFileBasedTransformerIllegalArguments() {
    // Test if the class throws illegal argument exception when argument not present.
    assertThrows(
        IllegalArgumentException.class,
        () -> sqlFileTransformer.apply(jsc, sparkSession, inputDatasetRows, props));
  }
```